### PR TITLE
refine pmml.py

### PR DESCRIPTION
**Pull request by wxchan**
and add it to check

### CODE REVIEW

File: pmml/pmml.py [hunk 1]
```diff
@@ -31,159 +29,98 @@ def get_threshold(node_id, prev_node_idx, is_child):
...
-    pmml_out = kwargs['out_file']
-    tab_len = kwargs['tab_length']
+def print_nodes_pmml(node_id, tab_len, is_left_child, prev_node_idx):
     if node_id < 0:
-        node_id = -1 * node_id
-        score = leaf_value[node_id - 1]
-        recordCount = leaf_count[node_id - 1]
+        node_id = ~node_id
+        score = leaf_value[node_id]
+        recordCount = leaf_count[node_id]
```
Comment (wxchan) [inline pmml/pmml.py]: I think `node_id = ~ node_id` should be same as `node_id = - node_id - 1` before? strange thing the model I generate seems a little different with this change

**Comment by wxchan** [comment]
travis check will not save 'model.txt' at test_basic.py?

**Comment by guolinke** [comment]
@wxchan I am not sure about it.
I think you can create an new folder, and put a small model into it, and run pmml test in this folder.

File: pmml/pmml.py (hunk 1 above)
Comment (guolinke) [inline pmml/pmml.py]: I think it should be same. Did you double check about this? maybe caused by other changes?

**Comment by wxchan** [comment]
look like it's ok now, I move it into basic.py with os.system

File: pmml/pmml.py (hunk 1 above)
Comment (wxchan) [inline pmml/pmml.py]: I found out the reason, node_id is also used in following codes.

**Comment by wxchan** [comment]
@guolinke this pr is finished too.

**Pull request merged**